.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Should we keep this line or should we maybe provide more context/documentation here?

Reply:
```suggestion
# This file allows you to configure your editor using [EditorConfig](https://EditorConfig.org).
# Make sure to install the EditorConfig plugin if your editor doesn't come with built-in support.
```

Reply:
@JKRhb you can commit if that is what you had in mind :)

Reply:
Looks good, thank you, @egekorkan! :)